Makefile

```diff
@@ -5,6 +5,7 @@ OUT ?= build
 BIN := $(OUT)/rv32emu
 
 CFLAGS = -std=gnu99 -O2 -Wall -Wextra
+CFLAGS += -fomit-frame-pointer -fno-stack-check -fno-stack-protector
```

Comment:
Move this line right after computed goto part. That is,
```
# Whether to enable computed goto
ENABLE_COMPUTED_GOTO ?= 1
...

# For tail-call elimination, we need a specific set of build flags applied.                                                                                   
$(OUT)/emulate.o: CFLAGS += -fomit-frame-pointer -fno-stack-check -fno-stack-protector
```

We shall carefully specify the compilation options.